projects/1.19/assets/heat-and-climate/dcs_climate/lang/zh_cn.json

```
@@ -0,0 +1,824 @@
+  "item.dcs_climate.food.string_tree":"树皮绳",
+  "item.dcs_climate.food.string_wool":"毛线",
+  "item.dcs_climate.food.string_cotton":"棉线",
+
+  "item.dcs_climate.food.dust_defatted_soy":"脱脂大豆",
+  "item.dcs_climate.food.dust_press_cake":"油渣",
+
+  "item.dcs_climate.food.animalfeed_hay":"干草",
+  "item.dcs_climate.food.animalfeed_straw":"稻草",
+  "item.dcs_climate.food.animalfeed_compound":"混合饲料",
+
+  "item.dcs_climate.food.fertilizer_mixed":"复合肥",
+
+  "item.dcs_climate.food.bread_round_raw":"面团",
+  "item.dcs_climate.food.bread_round_baked":"圆面包",
+  "item.dcs_climate.food.bread_square_raw":"方面团",
+  "item.dcs_climate.food.bread_square_baked":"吐司面包",
+  "item.dcs_climate.food.bread_nuts_raw":"坚果果干面团",
+  "item.dcs_climate.food.bread_nuts_baked":"坚果果干面包",
+  "item.dcs_climate.food.bread_cinnamon_raw":"肉桂卷面团",
+  "item.dcs_climate.food.bread_cinnamon_baked":"肉桂卷",
+  "item.dcs_climate.food.bread_anko_raw":"豆沙面团",
+  "item.dcs_climate.food.bread_anko_baked":"豆沙包",
+  "item.dcs_climate.food.bread_cream_raw":"奶油夹心面团",
+  "item.dcs_climate.food.bread_cream_baked":"奶油夹心面包",
+  "item.dcs_climate.food.bread_sausage_raw":"香肠面包面团",
+  "item.dcs_climate.food.bread_sausage_baked":"香肠面包",
+
+  "item.dcs_climate.food.sandwich_fruit":"果酱三明治",
+  "item.dcs_climate.food.sandwich_marmalade":"橘子三明治",
+  "item.dcs_climate.food.sandwich_egg":"鸡蛋三明治",
+  "item.dcs_climate.food.sandwich_salad":"沙拉三明治",
+  "item.dcs_climate.food.sandwich_salmon":"三文鱼三明治",
+
+  "item.dcs_climate.food.porridge_simple":"麦粥",
+  "item.dcs_climate.food.porridge_milk":"牛奶麦粥",
+  "item.dcs_climate.food.muesli":"牛奶麦片",
+
+  "item.dcs_climate.food.stick_beef_raw":"生牛肉串",
+  "item.dcs_climate.food.stick_beef_cooked":"烤牛肉串",
+  "item.dcs_climate.food.stick_pork_raw":"生猪肉串",
+  "item.dcs_climate.food.stick_pork_cooked":"烤猪肉串",
+  "item.dcs_climate.food.stick_mutton_raw":"生羊肉串",
+  "item.dcs_climate.food.stick_mutton_cooked":"烤羊肉串",
+  "item.dcs_climate.food.stick_chicken_raw":"生鸡肉串",
+  "item.dcs_climate.food.stick_chicken_cooked":"烤鸡肉串",
+  "item.dcs_climate.food.stick_offal_raw":"生内脏串",
+  "item.dcs_climate.food.stick_offal_cooked":"烤内脏串",
+
+  "item.dcs_climate.food.plate_beef_raw":"生牛排",
+  "item.dcs_climate.food.plate_beef_cooked":"烤牛排",
+  "item.dcs_climate.food.plate_meat_raw":"生肉排",
+  "item.dcs_climate.food.plate_meat_cooked":"烤肉排",
+  "item.dcs_climate.food.plate_legs_raw":"生腿排",
+  "item.dcs_climate.food.plate_legs_cooked":"烤腿排",
+  "item.dcs_climate.food.plate_big_garlic_raw":"生蒜香牛排",
+  "item.dcs_climate.food.plate_big_garlic_cooked":"烤蒜香牛排",
+  "item.dcs_climate.food.plate_big_chicken_raw":"生全鸡",
+  "item.dcs_climate.food.plate_big_chicken_cooked":"烤全鸡",
+  "item.dcs_climate.food.plate_fish_raw":"生鱼排",
+  "item.dcs_climate.food.plate_fish_cooked":"烤鱼排",
+
+  "item.dcs_climate.food.salad_green":"田园沙拉",
+  "item.dcs_climate.food.salad_potato":"土豆沙拉",
+  "item.dcs_climate.food.salad_nuts":"坚果豆类沙拉",
+  "item.dcs_climate.food.salad_melon":"西瓜沙拉",

```

```suggestion
  "item.dcs_climate.food.salad_melon":"西瓜番茄沙拉",
```
原文`スイカとトマトのサラダ`<s>什么黑暗料理</s>
但是居然真的是意式菜（[菜谱](https://cooking.nytimes.com/recipes/1014851-tomato-and-watermelon-salad)）